### DISCUSSION TITLE
EVM Revert error

### DISCUSSION THREAD

**Discussion by Jahanzeb9999** [Cyfrin/foundry-full-course-cu#1259] [Q&A]
[FAIL. Reason: EvmError: Revert] testPriceFeedVersionIsAccurate() (gas: 10172)
Traces:
  [10172] FundMeTest::testPriceFeedVersionIsAccurate() 
    ├─ [5136] FundMe::getVersion() [staticcall]
    │   ├─ [0] 0x0000000000000000000000000000000000000000::version() [staticcall]
    │   │   └─ ← ()
    │   └─ ← "EvmError: Revert"
    └─ ← "EvmError: Revert"
I already have passed in Sepolia rpc url .env file but still unable to depoly it to sepolia. Let me know guys!!

**Comment by altugoz** [comment]
What's the command you ran?
